rosbag2_storage_default_plugins/test/rosbag2_storage_default_plugins/sqlite/storage_test_fixture.hpp

```diff
@@ -73,48 +73,53 @@ class StorageTestFixture : public TemporaryDirectoryFixture
       });
 
     serialized_data->buffer_length = message_size;
-    int written_size = write_data_to_serialized_string_message(
+    size_t written_size = write_data_to_serialized_string_message(
```

Comment:
@wjwwood would 
```suggestion
    (void)write_data_to_serialized_string_message(
```
have the same effect?